File: 2650-split-with-minimum-sum/split-with-minimum-sum.py
class Solution:
    def splitNum(self, num: int) -> int:
        l = []
        for i in str(num):
            l.append(i)
        l.sort()

        minValue1 = ""
        minValue2 = ""

        for i in range(0, len(l), 2):
            minValue1 = minValue1 + l[i]
        for j in range(1, len(l), 2):
            minValue2 = minValue2 + l[j]
            
        return int(minValue1) + int(minValue2)

File: 2650-split-with-minimum-sum/split-with-minimum-sum.java
class Solution {
    public int splitNum(int num) {
        char[] digits = String.valueOf(num).toCharArray();
        Arrays.sort(digits);

        StringBuilder num1 = new StringBuilder();
        StringBuilder num2 = new StringBuilder();

        for (int i = 0; i < digits.length; i++) {
            if (i % 2 == 0) {
                num1.append(digits[i]);
            } else {
                num2.append(digits[i]);
            }
        }

        int minValue1 = Integer.parseInt(num1.toString());
        int minValue2 = Integer.parseInt(num2.toString());

        return minValue1 + minValue2;
    }
}
